testdata/baselines/reference/submoduleAccepted/compiler/argumentsReferenceInConstructor3_Js.types.diff

```diff
@@ -30,13 +29,12 @@
  		this.bar = super.arguments.foo;
 ->this.bar = super.arguments.foo : error
 ->this.bar : any
-+>this.bar = super.arguments.foo : object
++>this.bar = super.arguments.foo : any
```

Comment:
`any` is better than `object` because it shows that the sythetic cast is gone. There's an error, so `error` would be even better though.